Milestones.md

```diff
@@ -1,22 +0,0 @@
-#Milestones
```

Comment:
Removed, because we have working chat. I think further changes should be PRs as RFCs.